united-states-international-trade-commission.csl

```diff
@@ -154,11 +154,18 @@
           </choose>
         </else-if>
         <else-if type="broadcast motion_picture" match="any">
-          <names variable="director"/>
+          <names variable="director">
+            <name form="short" delimiter="" and="text" sort-separator=" "/>
```

Comment:
What's the idea of `delimiter=""` here? That seems wrong

Reply:
Good catch. The issue seems isolated to broadcasts/videos with 3 directors (which should be rare).  I'll change the code to `delimiter=", "`

Also, I need to make an adjustment to the contributor macro to fix name formatting in bibliographies.